docs/user-quick-start.md

```diff
@@ -0,0 +1,299 @@
+<!--
+SPDX-FileCopyrightText: The RamenDR authors
+SPDX-License-Identifier: Apache-2.0
+-->
+
+# Quick start guide for testing application DR
+
+This page will help you to set up an environment for testing **Regional-DR** basic tests.
+
+## What you’ll need
+
+To set up a *Ramen* development environment you will need to run 3
+Kubernetes clusters. *Ramen* makes this easy using minikube, but you need
+enough resources:
+
+- Bare metal or virtual machine with nested virtualization enabled
+- 8 CPUs or more
+- 20 GiB of free memory
+- 100 GiB of free space
+- Internet connection
+- Linux - tested on *Fedora* 37 and 38
+- non-root user with sudo privileges (all instructions are for non-root user)
+
+## Setting up your machine
+
+1. Fork the *Ramen* repository at https://github.com/RamenDR/ramen
+
+1. Clone the source locally
+
+   ```
+   git clone https://github.com/<your_github_id>/ramen.git
```

Comment:
we usually use `{variable}` - `<variable>` is more likely to break and cause trouble when tools try to escape it or when the shell treat it as redirection.